README.md

```diff
@@ -3,6 +3,7 @@
 
 [![npm version](https://badge.fury.io/js/auth0-deploy-cli.svg)](https://badge.fury.io/js/auth0-deploy-cli)
 [![CircleCI](https://circleci.com/gh/auth0/auth0-deploy-cli/tree/master.svg?style=svg)](https://circleci.com/gh/auth0/auth0-deploy-cli/tree/master)
+[![codecov](https://codecov.io/gh/auth0/auth0-deploy-cli/branch/master/graph/badge.svg?token=rlMcexLt6D)](https://codecov.io/gh/auth0/auth0-deploy-cli)
```

Comment:
We shouldn't need to put the token in here as this is a public repo. 🤔

Reply:
I suppose not but this is the link that Codecov provided officially 🤷‍♂️